packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -79,7 +79,7 @@ function Card(props: { propsToPass: any; type: string }) {
   }
 }
 
-function _CardContainer(props: CardContainerProps) {
+function CardContainerInternal(props: CardContainerProps) {
```

Comment:
New default rule requires that React components follow a naming convention that the leading underscore breaks.